config-service-impl/src/main/java/org/hypertrace/config/service/store/DocumentConfigStore.java

```diff
@@ -166,6 +175,23 @@ public ContextSpecificConfig getConfig(ConfigResourceContext configResourceConte
         .orElseGet(() -> emptyConfig(configResourceContext.getContext()));
   }
 
+  @Override
+  public Map<ConfigResourceContext, ContextSpecificConfig> getAllContextConfigs(
+      Set<ConfigResourceContext> configResourceContexts) throws IOException {
+    Map<ConfigResourceContext, Optional<ConfigDocument>> configDocs =
+        getLatestVersionConfigDocs(configResourceContexts);
+
+    return configDocs.entrySet().stream()
+        .collect(
+            Collectors.toUnmodifiableMap(
+                Entry::getKey,
+                entry ->
+                    entry
+                        .getValue()
+                        .flatMap(this::convertToContextSpecificConfig)
+                        .orElseGet(() -> emptyConfig(entry.getKey().getContext()))));
```

Comment:
Do you need to do this? Could you just return a map of the contexts that actually exist?

Reply:
can add filter let me try you can review then